sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1176,6 +1176,18 @@ def create_pipeline(self):
                 payload=b'2')))
 
 
+class FnApiRunnerTestWithGrpcAndMultiWorkers(FnApiRunnerTest):
+
+  def create_pipeline(self):
+    from apache_beam.options.pipeline_options import DirectOptions
+    p = beam.Pipeline(
+        runner=fn_api_runner.FnApiRunner(
+            default_environment=beam_runner_api_pb2.Environment(
+                urn=python_urns.EMBEDDED_PYTHON_GRPC)))
+    p._options.view_as(DirectOptions).direct_num_workers = 2
```

Comment:
Generally it's preferable to create a pipelines object and pass it to the Pipeline constructor instead of mutating it after the fact.